torch/testing/_internal/common_methods_invocations.py

```diff
@@ -9559,8 +9559,8 @@ def sample_inputs_max_unpool_grad(op_info, device, dtype, requires_grad, **kwarg
 
     ForeachFuncInfo(
         'floor',
-        dtypes=floating_types_and(torch.bfloat16),
-        dtypesIfCUDA=floating_types_and(torch.half, torch.bfloat16),
+        dtypes=all_types_and(torch.bfloat16),
```

Comment:
The regular OpInfos will need to be updated, and the floor, round, ceil, and trunc Python references in _refs/__init__py should be updated to reflect the new behavior, too

Reply:
Yup, updated the OpInfos. I think the `_refs` are alright for now (`ELEMENTWISE_TYPE_PROMOTION_KIND.DEFAULT` is used).